onboard/nav/stateMachine.hpp

```diff
@@ -99,6 +110,12 @@ class StateMachine
     // Number of waypoints completed.
     unsigned mCompletedWaypoints;
 
+    // Number of radio repeaters dropped.
+    unsigned mRadioRepeaterDrops;
```

Comment:
It doesn't seem like this variable does anything. Multiple other parts of our code base would need to change if we added multiple drops so I think we should remove this.